docker-compose.yml

```diff
@@ -2,7 +2,7 @@
 version: '2'
 services:
   zookeeper:
-    image: confluentinc/cp-zookeeper:latest
+    image: confluentinc/cp-zookeeper:4.1.0
```

Comment:
Because of Docker's counter-intuitive behavior for "latest", we should always specify the exact version of the image.